Makefile

```diff
@@ -67,6 +67,9 @@ delete:
 docker-buildx: check-env test
 	docker buildx build --platform=$(PLATFORM) -t $(IMAGE)-$(GOARCH) --build-arg BASE_IMAGE=$(BASE_IMAGE) --build-arg BUILD_IMAGE=$(BUILD_IMAGE) --build-arg $(GOARCH) --load .
 
+image:
```

Comment:
This target already exists above in the makefile.

Reply:
Fixed it. Thanks for catching